.github/workflows/nightly.yml

```
@@ -11,7 +11,7 @@ permissions:
 jobs:
   Win64:
     if: github.repository == 'darktable-org/darktable' || github.event_name == 'workflow_dispatch'
-    name: Nightly darktable.${{ github.sha }}.win64
+    name: Nightly darktable win64
```

I'd go even simpler and almost match ci.yml:

```suggestion
    name: Windows.UCRT64
```

"Nightly" is already prominent...

---

These regressed on purpose?